pom.xml

```diff
@@ -46,39 +46,9 @@
 
   <dependencies>
     <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-core</artifactId>
-      <version>2.12.0</version>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-databind</artifactId>
-      <version>2.12.0</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-collections4</artifactId>
-      <version>4.4</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-lang3</artifactId>
-      <version>3.11</version>
-    </dependency>
-    <dependency>
-      <groupId>commons-io</groupId>
-      <artifactId>commons-io</artifactId>
-      <version>2.6</version>
-    </dependency>
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-      <version>30.1-jre</version>
-    </dependency>
-    <dependency>
-      <groupId>javax.validation</groupId>
-      <artifactId>validation-api</artifactId>
-      <version>2.0.1.Final</version>
+      <groupId>org.folio</groupId>
```

Comment:
A release PR should only include the change for the release process.  The expectation is that any bugfixes have already been committed/merged prior to the release PR.  I was surprised to see these dependency adjustments here.

Reply:
Oh, OK I think I see what happened here.  Looks like you based this branch off master, not 1.0.1.  Since these dependency changes were applied to the master branch, you needed "undo" them for 1.0.2.  I would have been a lot cleaner/easier to just base this release branch off of v1.0.1.

Reply:
For additional information on hotfix and release branching strategies, see https://martinfowler.com/articles/branching-patterns.html#hotfix-branch